aggregate/central-site/pom.xml

```diff
@@ -8,7 +8,7 @@
     <version>4.3.0-SNAPSHOT</version>
   </parent>
   <artifactId>org.jboss.tools.site.central</artifactId>
-  <name>JBoss Tools - Central</name>
+  <name>JBoss Central</name>
```

Comment:
The contents are the same for these sites. You're thinking the DISCOVERY SITES, not the UPDATE SITES.